### PR TITLE
Feature/scenarios

### PR DESCRIPTION
**Pull request by marcinkp**

### CODE REVIEW

File: tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        </h2>
+        <p>
+          This template is served from the <strong>http mock</strong> repository.
+        </p>
+        <p>
+          This is <strong>1 snippet, 1 service</strong> example content.
+        </p>
+        <p>
+          <a class="btn btn-primary btn-large"
+             href="https://github.com/Cognifide/knotx">Learn more</a>
```
Comment (malaskowski) [inline tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html]: Please fix the URL to point to the performance-tests repo.

File: tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    limitations under the License.
+
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="http://cognifide.com/favicon.ico?v=2"/>
```
Comment (malaskowski) [inline tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html]: Let's get rid of it or change to `http://knotx.io/img/favicons/favicon-16x16.png`.

File: tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html (hunk 1 above)
Comment (marcinkp) [reply tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html]: fixed

File: tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html (hunk 2 above)
Comment (marcinkp) [reply tooling/mocks/content/public_html/scenario1/simple-1snippet-1service.html]: removed

**Review by malaskowski — APPROVED** [review]

**Pull request merged**